doc/setup/quickstart_cmake/index.md

```diff
@@ -1,3 +1,4 @@
 # Quick Setup (using CMake) {#setup}
 
-Please see the SeqAn documentation about [setup](https://docs.seqan.de/seqan/3-master-user/setup.html).
+Please see the SeqAn documentation about [setup](https://docs.seqan.de/seqan/3-master-user/setup.html), but note that
+Sharg has more strict software requirements than SeqAn! Sharg requires gcc ≥ 10 and cmake ≥ 3.10.
```

Comment:
I think these are pretty much the same as seqan3? seqan3 also requires gcc >= 10 (sharg dropped gcc9 before seqan3 did), and sharg should also work with cmake 3.4, unless you wanna build the tests, in which case both sharg and seqan3 need 3.10.

Reply:
SeqAn3 will be updated by https://github.com/seqan/seqan3/pull/3009